CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 ## 0.32.0
 
 * Dependency updates (JMX exporter 1.1.0)
+* Added support for the [Strimzi Metrics Reporter](https://github.com/strimzi/metrics-reporter) metrics.
+  This is a Kafka plugin that directly exports metrics in Prometheus format, and can be enabled by setting `bridge.metrics=strimziMetricsReporter`.
+
+### Changes, deprecations and removals
+
+* `KAFKA_BRIDGE_METRICS_ENABLED` has been deprecated.
+  The equivalent configuration is `bridge.metrics=jmxPrometheusExporter`. 
```

Comment:
"The equivalent configuration ..." is not that clear. I would state clearly that to keep the current behaviour and use JMX exporter, the property has to be set to `jmxPrometheusExporter`

Reply:
Ok, let me rephrase.